build.yaml

```diff
@@ -102,7 +102,22 @@ release:
                   find build -type f -name "*.${PACKAGE_TYPE}" -exec mv {} packages \; 
                   rm -rf build
                   PHP_TOKENS=($(echo ${PHP_VERSION} | tr "-" " "))
-                  sudo ${PACKAGE_UTILITY_REMOVE} ${PHP_TOKENS[0]}-common
+                  # Installing packaging and testing connection
+                  sudo ${PACKAGE_INSTALL} packages/${PHP_TOKENS[0]}*.${PACKAGE_TYPE}
```

Comment:
Overkill, but i would like to see a check for the module being loaded before the script is executed; this would also give you a short circuit mechanism as you would know that the module didn't load so therefore the PHP script won't work.

Reply:
Good idea. I'm facing an issue with module not loading on php-dse-driver. This would help identify such issues.